chemclipse/plugins/org.eclipse.chemclipse.ux.extension.xxd.ui/src/org/eclipse/chemclipse/ux/extension/xxd/ui/preferences/PreferenceConstants.java

```diff
@@ -778,6 +779,8 @@
 	public static final String DEF_PCR_DEFAULT_COLOR = "192,192,192";
 	public static final String P_PCR_COLOR_CODES = "pcrColorCodes";
 	public static final String DEF_PCR_COLOR_CODES = "";
+	public static final String P_PCR_REFERENCE_LABEL = "pcrReferenceLabel";
+	public static final String DEF_PCR_REFERENCE_LABEL = LabelSetting.COORDINATE_SAMPLENAME.getLabel();
```

Comment:
I would recommend to persist "LabelSetting.COORDINATE_SAMPLENAME.name()". The label itself if variable and could be translated in the future. The enum entry name itself is independent of a possibly translated label.